cmd/cogs/main.go

```diff
@@ -2,8 +2,29 @@ package main
 
 import (
 	"fmt"
+	"os"
+
+	"github.com/bestowinc/cogs"
+	"github.com/docopt/docopt-go"
 )
 
 func main() {
-	fmt.Println("Hello configs!")
+	usage := `Usage:
+  example generate <env> <cog-file>`
+
+	opts, _ := docopt.ParseArgs(usage, os.Args[1:], "0.1")
+	var conf struct {
+		Generate bool
+		Env      string
+		File     string `docopt:"<cog-file>"`
+	}
+
+	opts.Bind(&conf)
+	_ = fmt.Println
```

Comment:
curious on the reasoning of this println

Reply:
whoops, forgot to leave it out.